src/components/rating/rating.tsx

```diff
@@ -101,6 +121,7 @@ export class Rating
   connectedCallback(): void {
     connectLabel(this);
     connectForm(this);
+    this.renderArray = Array.from({ length: this.max }, (_, i) => i + 1);
```

Comment:
nitpick: maybe this array could be called `renderStars`?

Reply:
I've moved this.max to private vars so this isn't an issue. I can update the name to be more descriptive.